homeassistant/components/sensibo/coordinator.py

```diff
@@ -167,6 +167,12 @@ async def _async_update_data(self) -> SensiboData:
                         rssi=measurement.get("rssi"),
                     )
 
+            # Add information for pure devices
```

Comment:
The data processing is getting a bit complex here.  It would be better to have all of this in the library and present everything as a dataclass to Home Assistant

Reply:
The drawback of that would be that any new addition of data would require both a change in the package and the integration.
In terms of processing it would be the same I guess?

So essentially move the processing into a data class in the package to have cleaner code here in the coordinator?

I'm not against but would prefer to merge the two open PR's first and then stop new ones and do the "move" upstream.

Reply:
> So essentially move the processing into a data class in the package to have cleaner code here in the coordinator?

Yes, also the review on the Home Assistant side will be smaller each time which means PRs can move faster on this side.

Reply:
> I'm not against but would prefer to merge the two open PR's first and then stop new ones and do the "move" upstream.

WFM 👍